src/main/resources/assets/create/lang/zh_cn.json

```
@@ -2384,11 +2873,96 @@
 	"create.ponder.stressometer.text_3": "红石比较器可以根据应力表的数值输出不同强弱的红石信号",
 
 	"create.ponder.super_glue.header": "使用强力胶来黏附方块",
-	"create.ponder.super_glue.text_1": "强力胶可以在任意两个方块间使用",
-	"create.ponder.super_glue.text_2": "当被粘合的方块被组装为装置时，他们会一起运动",
-	"create.ponder.super_glue.text_3": "当强力胶在副手时……",
-	"create.ponder.super_glue.text_4": "……新放置的方块会自动被黏附在所放置方块的面上",
-	"create.ponder.super_glue.text_5": "左击可以清除强力胶",
+	"create.ponder.super_glue.text_1": "强力胶用于将方块组合成移动装置",
+	"create.ponder.super_glue.text_2": "右击两个端点来创建一个新的“胶合”区域",
+	"create.ponder.super_glue.text_3": "手持强力胶左击可以移除选区",
+	"create.ponder.super_glue.text_4": "同一个选区内相邻的方块会互相拉动对方",
+	"create.ponder.super_glue.text_5": "互相重叠的选区会一起移动",
+	"create.ponder.super_glue.text_6": "悬挂在其它方块上的方块通常不需要强力胶",
+
+	"create.ponder.track_chunks.header": "横穿卸载的区块",
+	"create.ponder.track_chunks.text_1": "轨道在加载的区块之外仍然保持运作",
+	"create.ponder.track_chunks.text_2": "列车将毫无问题地横穿世界上不活跃的区块",
+	"create.ponder.track_chunks.text_3": "他们仍然会停在车站或红灯处",
+	"create.ponder.track_chunks.text_4": "但是，钻头和其他机载机器将无法运行",
+	"create.ponder.track_chunks.text_5": "一旦靠近玩家，列车将重新出现",
+
+	"create.ponder.track_observer.header": "侦测列车",
+	"create.ponder.track_observer.text_1": "选择一个列车轨道，然后将侦测器放在附近",
+	"create.ponder.track_observer.text_2": "侦测器将检测任何经过标记的列车",
+	"create.ponder.track_observer.text_3": "侦测器可以过滤为仅给匹配的货物激活",
+
+	"create.ponder.track_placement.header": "放置列车轨道",
+	"create.ponder.track_placement.text_1": "一种专为列车装置设计的新型轨道",
+	"create.ponder.track_placement.text_2": "要批量放置多排轨道，单击现有轨道",
+	"create.ponder.track_placement.text_3": "然后选择或放置第二个轨道",
+	"create.ponder.track_placement.text_4": "轨道也可以放置为转弯或斜坡",
+	"create.ponder.track_placement.text_5": "连接时，轨道将尝试使每个转弯的大小相等",
+	"create.ponder.track_placement.text_6": "连接时按住冲刺键……",
+	"create.ponder.track_placement.text_7": "……将创建最长的适合的弯道",
+	"create.ponder.track_placement.text_8": "副手的材料会自动铺在轨道下",
+
+	"create.ponder.track_portal.header": "轨道与下界",
+	"create.ponder.track_portal.text_1": "对着下界传送门放置的轨道……",
+	"create.ponder.track_portal.text_2": "……将尝试在另一侧创建成对的轨道",
+	"create.ponder.track_portal.text_3": "这条轨道上的列车现在可以跨维度旅行",
+
+	"create.ponder.train_assembly.header": "组装列车",
+	"create.ponder.train_assembly.text_1": "选择列车轨道，然后将车站放置在附近",
+	"create.ponder.train_assembly.text_10": "每列列车都需要装上列车驾驶台",
+	"create.ponder.train_assembly.text_11": "可选的第二个允许从车站往两个方向出发",
+	"create.ponder.train_assembly.text_12": "打开车站UI并确认组装过程",
+	"create.ponder.train_assembly.text_13": "列车只能在车站被拆解为方块",
+	"create.ponder.train_assembly.text_14": "在车站上使用时，地图会在该位置添加一个带标签的标记",
+	"create.ponder.train_assembly.text_15": "组装好的列车可以使用扳手重新安置到相邻的轨道",
+	"create.ponder.train_assembly.text_2": "车站是你的轨道网络的路径点",
+	"create.ponder.train_assembly.text_3": "要创建新列车，打开UI并切换到组装模式",
+	"create.ponder.train_assembly.text_4": "在组装期间，时刻表上的列车不会接近这个车站",
+	"create.ponder.train_assembly.text_5": "在轨道上使用列车机壳创建新的转向架",
+	"create.ponder.train_assembly.text_6": "再次单击轨道以在转向架设计之间循环",
```

```suggestion
	"create.ponder.train_assembly.text_6": "再次单击轨道以在转向架类型之间切换",
```
新人刚用github，问下怎么查看英文原材料（方便校对

---

https://github.com/Creators-of-Create/Create/blob/mc1.18/dev/src/generated/resources/assets/create/lang/en_us.json

---

```suggestion
	"create.ponder.train_assembly.text_6": "再次单击轨道，便可以在在转向架设计之间循环",
```
是否是“设计”仍存疑。

---

我认为“设计”、“样式”确实没法囊括，应当改为“类型”

```suggestion
	"create.ponder.train_assembly.text_6": "再次单击轨道，便可以在转向架类型之间循环",
```